inc/saitunnel.h

```diff
@@ -325,6 +325,22 @@ typedef enum _sai_tunnel_type_t
 
 } sai_tunnel_type_t;
 
+/**
+ * @brief Defines VXLAN tunnel UDP source port mode
+ */
+typedef enum _sai_tunnel_vxlan_udp_sport_mode_t
+{
+    /**
+     * @brief User define value
+     */
+    SAI_TUNNEL_VXLAN_UDP_SPORT_MODE_USER_DEFINED,
+
+    /**
+     * @brief RFC6335 Computed hash value in range 49152-65535
+     */
+    SAI_TUNNEL_VXLAN_UDP_SPORT_MODE_EPHEMERAL,
```

Comment:
this is another way to keep the enum values compatible with the latter version than v1.7, i left them there